public/css/style.css

```diff
@@ -1,270 +1,340 @@
-  background-color: rgb(0, 248, 131.75);
+    background-color: rgb(0, 248, 131.75);
 }
 
 .single-applicant .applicant-header {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
 }
 
 .single-applicant .details h2 {
-  color: #743DFB;
+    color: #743DFB;
 }
 
 .single-applicant .details p span {
-  font-weight: bold;
+    font-weight: bold;
+}
+
+.applicationInfo {
+
 }
 
 .login {
-  margin-top: 10px;
+    margin-top: 10px;
 }
 
 .login section {
-  width: 100%;
-  max-width: 500px;
-  margin: 0 auto;
-  padding: 15px;
-  border-radius: 10px;
-  background-color: #dddddd;
+    width: 100%;
+    max-width: 500px;
+    margin: 0 auto;
+    padding: 15px;
+    border-radius: 10px;
+    background-color: #dddddd;
 }
 
 .login section form label {
-  display: block;
-  margin-bottom: 5px;
-  font-weight: bold;
+    display: block;
+    margin-bottom: 5px;
+    font-weight: bold;
 }
 
 .login section form input[type=email], .login section form input[type=password] {
-  width: 100%;
-  margin-bottom: 15px;
+    width: 100%;
+    margin-bottom: 15px;
 }
 
 .login section form .invalid-login {
-  margin-top: 10px;
+    margin-top: 10px;
 }
 
 .admin .links {
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr 1fr;
-  gap: 20px;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    gap: 20px;
 }
 
 .admin .links a {
-  padding: 50px 20px;
-  border: 2px solid #337ab7;
-  border-radius: 5px;
-  text-align: center;
+    padding: 50px 20px;
+    border: 2px solid #337ab7;
+    border-radius: 5px;
+    text-align: center;
 }
 
 .admin .links a:hover {
-  background-color: #dddddd;
-  color: #337ab7;
+    background-color: #dddddd;
+    color: #337ab7;
 }
 
-@media screen and (max-width: 768px) {
-  header {
-    flex-direction: column-reverse;
-  }
+.archived {
+    color: #FF0000;
+    font-size: 2rem;
+}
 
-  .admin .links {
-    grid-template-columns: 1fr;
-  }
+@media screen and (max-width: 768px) {
+    header {
+        flex-direction: column-reverse;
+    }
+
+    .admin .links {
+        grid-template-columns: 1fr;
+    }
+}
+
+
+td:has(> .edit-button) {
+    position: relative;
+}
+
+.edit-button {
+    top: 50%;
+    transform: translateY(-50%);
+    padding: 6px;
+    border: 1px solid #dddddd;
+    border-radius: 10px;
+    font-size: 14px;
+    position: absolute;
+    background-color: #61fed2;
+    color: black;
+    transition: all 0.1s linear;
+    box-shadow: 0 0 0 0 rgb(0,0,0);
+}
+.edit-button:hover {
+    background-color: #dddddd;
+}
+
+.archive-button {
+    left: 50%;
+    transform: translateY(-50%);
+    padding: 6px;
+    border: 1px solid #dddddd;;
+    border-radius: 10px;
+    font-size: 14px;
+    position: absolute;
+    background-color: #61fed2;
+    color: black;
+    transition: all 0.1s linear;
+    box-shadow: 0 0 0 0 rgb(0,0,0);
+}
+.archive-button:hover {
+    background-color: #dddddd;
 }
```

Comment:
Remove these CSS classes, they're redundant

Reply:
ill do this